Remix/docset.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Remix",
+    "version": "1.9.0",
+    "archive": "remix.tgz",
```

Comment:
This file doesn't exist. You added `Remix.tgz` instead (note: case-sensitive).

Reply:
Fixed - and the docset isn't in the `docsets` directory either 👍